### PR TITLE
s.to Fix

### PR DESCRIPTION
**Pull request by MauriceX24**
- Removed iFrame
- Removed "sourceMap"
- Removed "board.s.to"

Image

### CODE REVIEW

**Comment by MauriceX24** [comment]
Uff ...sorry for this mistake :o 
Mistake

File: s.to/dist/presence.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
+var presence = new Presence({
+    clientId: "463000750193246209",
+    mediaKeys: false
```
Comment (Bas950) [inline s.to/dist/presence.js:3]: remove media keys

File: s.to/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "version": "1.5.0",
+  "logo": "https://i.imgur.com/AJbaeCC.png",
+  "thumbnail": "https://i.imgur.com/L0Zqja4.png",
+  "color": "#0000ff",
+  "tags": [
+    "video",
+    "movie",
+    "serie"
+  ],
+  "iframe": true,
```
Comment (Bas950) [inline s.to/dist/metadata.json:25]: remove iframe line since you dont use iframes...

**Comment by doomlerd** [comment]
Fix the deepscan errors before we can merge.

File: s.to/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,298 @@
+var presence = new Presence({
+    clientId: "463000750193246209",
+}), strings = presence.getStrings({
```
Comment (Bas950) [inline s.to/dist/presence.js:3]: remove comma after client id line, and fix deepscan errors

Comment (doomlerd) [reply s.to/dist/presence.js:3]: Update your fork @MauriceX24 to fix the deepscan errors.

**Comment by doomlerd** [comment]
Open another PR and only change the files you need to change, not everything.

**Pull request closed without merging**